src/test/system_tests/ceph_s3_tests/s3-tests-lists/s3_tests_pending_list.txt

```diff
@@ -15,6 +15,13 @@ s3tests_boto3.functional.test_s3.test_post_object_missing_policy_condition
 s3tests_boto3.functional.test_s3.test_post_object_request_missing_policy_specified_field
 s3tests_boto3.functional.test_s3.test_post_object_expired_policy
 s3tests_boto3.functional.test_s3.test_post_object_invalid_request_field_value
+s3tests_boto3.functional.test_s3.test_lifecycle_get_no_id
+s3tests_boto3.functional.test_s3.test_object_lock_put_obj_lock_invalid_mode
+s3tests_boto3.functional.test_s3.test_object_lock_get_obj_retention_iso8601
+s3tests_boto3.functional.test_s3.test_object_lock_delete_object_with_retention_and_marker
+s3tests_boto3.functional.test_s3.test_multipart_upload_on_a_bucket_with_policy
+s3tests_boto3.functional.test_s3.test_multipart_upload_small
+s3tests_boto3.functional.test_s3.test_post_object_authenticated_request_bad_access_key
```

Comment:
@nimrod-becker, just want to make sure that these tests should not be in the `s3_tests_black_list` (specially all the tests with the prefix `object_lock_`)